cpp/lib/src/Zotero.cc

```diff
@@ -703,6 +703,17 @@ void MarcFormatHandler::generateMarcRecord(MARC::Record * const record, const st
 
     if (not node_parameters.journal_name_.empty())
         record->insertField("JOU", { { 'a', node_parameters.journal_name_ } });
+
+    // remove any fields that match removal patterns
+    for (const auto &filter : site_params_->field_removal_filters_) {
+        auto field(filter.first);
+        if (record->fieldOrSubfieldMatched(filter.first, filter.second.get())) {
+            if (field.length() != 3)
```

Comment:
This test belongs to the place where we load the config and then we should probably abort hard if we don't meet our requirement.  (Also maybe we should also support subfields like 520a etc.)